__tests__/components/AnonymousChat/child/ChatItemTargetText.test.tsx

```diff
@@ -34,7 +40,8 @@ const renderChatItemTargetText = (isContinous = false) =>
     chatType: SIGNED,
     data: expectedReplyPreview.payload,
     isContinuous: isContinous,
-    time: timestamp
+    time: timestamp,
+    attachments: []
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `attachments` field is added to the chat item object. Ensure that all functions and components using this object are updated to handle this new field correctly. If there's a possibility of not having any attachments, consider making this field optional or ensure it defaults to an empty array if no attachments are provided.
```diff
-    time: timestamp
+    time: timestamp,
+    attachments: []
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->